ISLE/isle.cpp

```diff
@@ -39,7 +39,8 @@ Isle::Isle()
   rect.m_right = 639;
   rect.m_bottom = 479;
 
-  m_videoParam = MxVideoParam(rect, NULL, 1, MxVideoParamFlags());
+  MxVideoParamFlags video_flags;
+  m_videoParam = MxVideoParam(rect, NULL, 1, video_flags);
```

Comment:
While it's true that GCC won't compile with the original, this produces less accurate code in MSVC420. Accuracy is the priority with this project, modernizations should be kept to other repos/forks.

Reply:
Aight, revert it will be.

Reply:
I've rebased on current master, and force pushed it.
If you prefer reverts to avoid re-reviews, let it know.

Reply:
About accuracy, both implementations create the flags on the stack, and the variable is not used afterwards.
So the compiler should be able to emit identical assembly.